docs/src/operations.md

```diff
@@ -296,7 +296,7 @@ conditional_∫c = Field(Integral(c, dims=(1, 2), condition=cond)) # integrate o
 ├── operand: Integral of ConditionalOperation of BinaryOperation at (Center, Center, Center) with condition cond (generic function with 1 method) over dims (1, 2)
 ├── status: time=0.0
 └── data: 1×1×11 OffsetArray(::Array{Float64, 3}, 1:1, 1:1, -2:8) with eltype Float64 with indices 1:1×1:1×-2:8
-    └── max=0.0, min=0.0, mean=0.0
+    └── max=1.27516e14, min=1.27516e14, mean=1.27516e14
 ```
 
 Above we have attached a condition to the operand.
```

Comment:
we need to change the text here because it reads as if the field is not yet computed
also need to remove the `compute!(conditional_∫c)` from below?